HelpSource/Classes/AbstractWrappingDispatcher.schelp

```diff
@@ -1,60 +1,85 @@
-CLASS:: AbstractWrappingDispatcher
-summary:: A dispatcher which wraps functions to support multiple paramter matching
+title:: AbstractWrappingDispatcher
+summary:: a dispatcher which wraps functions to support multiple paramter matching
 categories:: External Control>Abstract Classes
 related:: Classes/AbstractDispatcher, Classes/AbstractMessageMatcher, Classes/OSCMessageDispatcher, Classes/OSCMessagePatternDispatcher, Classes/MIDIMessageDispatcher, Classes/OSCFunc, Classes/OSCdef, Classes/MIDIFunc, Classes/MIDIdef, Classes/AbstractResponderFunc
 
-DESCRIPTION::
+
+description::
+
+
 AbstractWrappingDispatcher extends AbstractDispatcher to provide the facility to wrap response functions in specialised objects (instances of subclasses of link::Classes/AbstractMessageMatcher:: to efficiently support matching of multiple parameters. Its subclasses link::Classes/OSCMessageDispatcher:: and link::Classes/MIDIMessageDispatcher:: match using a flat dictionary lookup of the 'most significant' parameter, and only attempt to match other parameters if an initial match is found. This approach is faster than others (such as multi-level dictionaries) for most configurations.
 
 
-CLASSMETHODS::
+classmethods::
+
+
+instancemethods::
 
 
-INSTANCEMETHODS::
 private:: init
 
-METHOD:: wrappedFuncs
-Get a dictionary of all currently wrapped functions, stored using their owning responder funcs as keys.
 
-returns:: An link::Classes/IdentityDictionary::.
+method::wrappedFuncs
+
+gets a dictionary of all currently wrapped functions, stored using their owning responder funcs as keys.
+
+returns::
+
+an link::Classes/IdentityDictionary::.
+
+
+method::add
+
+adds a responder func to this dispatcher. Subclasses should extend this to do any necessary bookkeeping. Generally this method should add this dispatcher as a dependant of the responder func, so that it can respond to any changes.
```

Comment:
The classic helpfiles in supercollider mostly used the "instructive tone" (something like "add a responder func") instead of the "desctiptive tone" ("adds a respinder func").

I liked the instructive tone and I don't see a reason we should change it.

Reply:
I thought 'descriptive tone' was used more now. Either my memory is wrong or I didn't have a big enough sample.
I may have chosen the wrong verbs.

Reply:
No, the verbs are ok. It is a general question, because this is a large and general edit. So it makes sense to ask what the standard should be.